src/Ionide.ProjInfo.ProjectSystem/ProjectSystem.fs

```diff
@@ -36,7 +36,7 @@ type ProjectResponse =
 /// Public API for any operations related to workspace and projects.
 /// Internally keeps all the information related to project files in current workspace.
 /// It's responsible for refreshing and caching - should be used only as source of information and public API
-type ProjectController(toolsPath: ToolsPath) as x =
+type ProjectController(toolsPath: ToolsPath, workspaceLoaderFactory: ToolsPath -> IWorkspaceLoader) as x =
```

Comment:
This now takes a function to create a workspace loader.  Allowing us to keep the current Loader but letting people trial the new one.

Reply:
I really like this 👍